sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpExclusionsFileFilter.java

```diff
@@ -75,7 +74,7 @@ public boolean accept(InputFile inputFile) {
    * (i.e., it starts with <code>"//"</code>).
    */
   private static class AverageLineLengthCalculator {
-    private static final Predicate<String> PHP_OPEN_TAG = Pattern.compile(LexicalConstant.PHP_START_TAG).asPredicate();
+    private static final Pattern PHP_OPEN_TAG = Pattern.compile(LexicalConstant.PHP_OPENING_TAG);
```

Comment:
The replacement of `PHP_START_TAG` with `PHP_OPENING_TAG` is the real change, the rest has no impact. The title of the PR is therefore misleading.